### ISSUE TITLE
EPD Ring Matrix

### ISSUE THREAD

**Issue by tlprotzman** [Skagamaster/Python_Pico_Reader#9]
It appears that only the first 16 elements of the ring sum matrix are being filled by [EPD_Hits.generate_epd_hit_matrix](https://github.com/Skagamaster/Python_Pico_Reader/blob/main/pico_reader/pico_reader.py#L76).  Is it intended that they will be split by east/west (i.e. east hits in the first 16 slots, west hits in the second 16)?

**Comment by Skagamaster** [comment]
You're right; there are only rings 1-16. EW must be specified. I'll check it out.

**Comment by Skagamaster** [comment]
OK, this should be fixed. It still needs to be tested; I'll put it in a pull request until it's tested.
